src/lib/models/keyframe.js

```diff
@@ -184,7 +201,7 @@ module.exports = class Keyframe {
                     },
                     volumes: {
                       type: 'array',
-                      minItems: 0,
```

Comment:
Why are volumes required?

Reply:
they aren't, but It looks clearer to me, that if `volumes:` section is used, it should declare at least one volume. Otherwise, it can be just skipped. 
Should we support this syntax `volumes: {}` ?

Reply:
Oh, okay. I assumed that if the minItems was set, than you had to have the parent key. If not, then I agree with you.